tests/ha/conftest.py

```diff
@@ -23,3 +29,45 @@ def setup_SmartSwitchHaTrafficTest(duthost, ptfhost, ptfadapter, vmhost, tbinfo)
     ha_io = SmartSwitchHaTrafficTest(activehost, standbyhost, ptfhost,
                                      ptfadapter, vmhost, tbinfo, io_ready, namespace="ns1")
     return ha_io
+
+
+@pytest.fixture(scope="module")
+def setup_namespaces_with_routes(ptfhost, duthosts):
+    """
+    Set up 4 PTF namespaces (ns1, ns2, ns3, ns4), assign interfaces from both DUTs,
+    configure IPs, static routes on PTF & DUTs, and clean up after the module.
+
+    Each DUT port goes into a unique namespace for full isolation.
+    """
+
+    # Namespace and interface mapping
+    ns_ifaces = [
+        {"namespace": "ns1", "iface": "eth3", "ip": "172.16.1.1/24", "dut": duthosts[0], "next_hop": "172.16.1.254"},
+        {"namespace": "ns2", "iface": "eth4", "ip": "172.16.2.1/24", "dut": duthosts[0], "next_hop": "172.16.2.254"},
+        {"namespace": "ns3", "iface": "eth22", "ip": "172.16.3.1/24", "dut": duthosts[1], "next_hop": "172.16.3.254"},
+        {"namespace": "ns4", "iface": "eth23", "ip": "172.16.4.1/24", "dut": duthosts[1], "next_hop": "172.16.4.254"},
```

Comment:
Per offline discussion, please simplify these by:
1. create two namespaces only
2. move the hard-coded string out of the setup fixture
3. ensure IP assigned to interfaces in the same name space are under same subnet